### DISCUSSION TITLE
验证频繁

### DISCUSSION THREAD

**Discussion by xzg178502** [liuyunfz/chaoxing_tool#87] [General]
![image](https://user-images.githubusercontent.com/121687267/233319435-c0c03b30-26a1-4bc1-9a17-2c13ad02a9cd.png)

**Comment by liuyunfz** [comment]
这个我倒是第一次遇到，个人之前没碰到过
请问最近还有出现该情况吗，有的话麻烦提供一些更多信息以复现
